docs/upgrade/interchangeable-filtering.md

```diff
@@ -0,0 +1,79 @@
+# Upgrade Instructions for Interchangeable Filtering
+
+This article describes upgrade instructions for [#943 Elastic search filtering](https://github.com/shopsys/shopsys/pull/943).
+Upgrade instructions are in a separate article because there is a lot of instructions and we don't want to jam the [UPGRADE-unreleased.md](/docs/upgrade/UPGRADE-unreleased.md). <!--- TODO change to released version --->
```

Comment:
The comment part is visible in docs... It shouldn't...

Reply:
fixed